woodchipper/http/flask.py

```diff
@@ -24,6 +24,12 @@ def wrapped_full_dispatch_request(self):
                 "body_size": request.content_length,
                 "method": request.method,
                 "url": request.url,
```

Comment:
Let's change this to `path` and extract `request.base_url` as its value?

Reply:
Done in rebase 👍 

Added some tests to confirm it's only pulling the path as well and not the query string.